README.md

```diff
@@ -62,46 +57,92 @@ store.addListener() {
 
 ## Side Effects
 
-Updating the application's state using pure functions is nice and it has a lot of benefits. Applications have to deal with the external world though (e.g., API call, disk files management, …). For all this kind of operations, Katana provides the concept of  `side effects`. Side effects can be used to interact with other part of your applications and then dispatch new actions to update your state.
+Updating the application's state using pure functions is nice and it has a lot of benefits. Applications have to deal with the external world though (e.g., API call, disk files management, …). For all this kind of operations, Katana provides the concept of  `side effects`. Side effects can be used to interact with other part of your applications and then dispatch new `StateUpdater` to update your state. For more complex situations, you can also dispatch other `side effects`.
```

Comment:
Updating the application's state using pure functions is nice and it has a lot of benefits. In the real world tho, applications have to deal with external entities (e.g., API call, disk files management, …).

Reply:
`with other part of your applications` -> `with other partS of your application`

Reply:
and then dispatch new StateUpdater `->` and then dispatch new `StateUpdater`s